docs/tour/hello-world.mdx

```diff
@@ -45,12 +86,17 @@ This command bootstraps 3 independent processes to create the necessary local in
 By default, `wash` will run the host in interactive mode.
 If you'd rather run the host in the background, run `wash up --detached` (or `wash up -d`, for short).
 
-## Generating a New Project
+## Creating a Project
 
-Now that our wasmCloud host is running, let's generate a new actor project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
+Now that our wasmCloud host infrastructure is running, let's generate a new actor component project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
 
-This command generates a new actor project in the `./hello` directory:
+Pick your language of choice below to follow 
 
+
+<Tabs groupId="lang" queryString>
+  <TabItem value="rustmod" label="Rust" default>
```

Comment:
Nit: I don't think new users will know the difference between "Rust" and "Rust Component"

Reply:
Do you think it's confusing enough to remove the old example entirely? I didn't like it either, and the difference between a module and a component in this case is really just the SDK you use to write the Wasm. I felt that it was odd to rework the docs to use the component examples and still mark them as experimental when it comes to possibly changing WASI APIs

Reply:
Yeah, we're in a bit of a bind for now. How about `Rust (Module)` and `Rust (Component)`?

Reply:
If you wanted to formalize the use of these, you could create a component that re-exports the tabs with no access to change the props. Usage example below:

```mdx
<LangTabs>
  <RustTab>...</RustTab>
  <RustTab component>...</RustTab>
  <GoTab>...</GoTab>
  <JSTab>...</JSTab>
  <PythonTab>...</PythonTab>
  ...etc
</LangTabs>
```

Reply:
If you want to do this, let me know and I can assist. There's a way to export components so they don't need to be imported in every file, too.

Reply:
I don't see it as a blocker for this PR but I'd love to implement this. Same thing with OS/platform things. @lachieh would you mind filing an enhancement issue?

Reply:
[Done](https://github.com/wasmCloud/wasmcloud.com/issues/254)